doc/internal/man3/ossl_rcu_lock_new.pod

```diff
@@ -0,0 +1,255 @@
+=pod
+
+=head1 NAME
+
+ossl_rcu_lock_new,
+ossl_rcu_lock_free, ossl_rcu_read_lock,
+ossl_rcu_read_unlock, ossl_rcu_write_lock,
+ossl_rcu_write_unlock, ossl_synchronize_rcu,
+ossl_rcu_call, ossl_rcu_derefrence,
+ossl_rcu_assign_pointer
+- perform read-copy-update locking
+
+=head1 SYNOPSIS
+
+ CRYPTO_RCU_LOCK *ossl_rcu_lock_new(int num_writers);
```

Comment:
Should the argument be unsigned?

Reply:
Right now, no, but its documented as needing to be at least 1, and errors out if the value is less than that.  I can't think of any reason why we might want a negative value here, but we certainly don't need a huge positive on, and so I'd like to keep it signed in case a negative value becomes useful in the future